WorkbenchProject/src/main/java/edu/uwb/braingrid/workbench/provvisualizer/controller/ProVisCtrl.java

```diff
@@ -7,6 +7,8 @@
 import java.util.HashMap;
 import java.util.ArrayList;
 import java.util.Scanner;
+
+import edu.uwb.braingrid.workbenchdashboard.WorkbenchDisplay;
```

Comment:
This import should go with the second group of import statements (project imports) and in alphabetical order within that group.

Reply:
Right, I think IntelliJ just threw it wherever it wanted, I'll put it where it should be.

Reply:
Please also remove the blank line that was inserted. Everything else looks good.